<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>taxistssssDialog</class>
 <widget class="QDialog" name="taxistssssDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>761</width>
     <height>291</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>№</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ФИО</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Телефон</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Модель машины</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Номер машины</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Цвет машины</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Репутация</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButtonCreate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelete">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
